### PR TITLE
feat: offline backends - users metadata (AR-3124)

### PR DESCRIPTION
**Pull request by yamilmedina**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-3124" title="AR-3124" target="_blank">TaskAR-3124</a>  [Android] Implement UX for missing conversation and user metadata
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [x] contains a reference JIRA issue number like `SQPIT-764`
    - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

We need to support displaying/ hiding and restrict options for users that we don't have metadata for.
This is done in, other user profile, participants lists and conversations list.

### Solution

If a backend is down, and we only have ids of these remote users we present a better user experience.

### Dependencies (Optional)

https://github.com/wireapp/kalium/pull/1698

Needs releases with:

- [x] GitHub link to other pull request

#### Test Coverage (Optional)

- [ ] I have added automated test to this contribution

### Attachments (Optional)

----
#### PR Post Submission Checklist for internal contributors (Optional)

- [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

- [x] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-1758/1/) **failed**.

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-1758/2/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.2.0(6431527)(6431527)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.2.0(6431527)(6431527)-staging-compat.apk)

File: app/src/main/kotlin/com/wire/android/ui/userprofile/other/OtherUserDevicesScreen.kt [hunk 1]
```diff
@@ -74,7 +109,7 @@ fun OtherUserDevicesScreen(
                             tag = "learn_more",
                             annotation = supportUrl,
                             onClick = {
-                                CustomTabsHelper.launchUrl(context, supportUrl)
+                                com.wire.android.util.CustomTabsHelper.launchUrl(context, supportUrl)
```
Comment (alexandreferris) [inline app/src/main/kotlin/com/wire/android/ui/userprofile/other/OtherUserDevicesScreen.kt]: maaaybe we can remove the import here

**Review by alexandreferris — APPROVED** [review]
Great work! Just have a tiny comment

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-1758/3/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.2.0(6467514)(6467514)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.2.0(6467514)(6467514)-staging-compat.apk)

**Comment by AndroidBob** [comment]
Build [4](https://10.10.124.134/job/wire-android-reloaded/job/PR-1758/4/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.2.0(6474508)(6474508)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.2.0(6474508)(6474508)-staging-compat.apk)

**Pull request merged**